spec/safe_commit_spec.rb

```diff
@@ -2,10 +2,6 @@
 
 RSpec.describe SafeCommit do
```

Comment:
⚠️ Think we should add more specs for the methods in `SafeCommit`? 🤔

Reply:
thanks. I've added them now